client/EditContributionView.js

```diff
@@ -83,7 +111,10 @@ class EditContributionView extends Component {
             contribBio: '',
             mediaProcess: '',
             contentEditing: '',
-            contributionData: null
+            contributionData: null,
+            approvalSwitch: false,
+            publishedSwitch: false,
+            publishedList: null,
```

Comment:
don't need this in state for both child components, use syncDoc in App.js and pass down via props